s3/src/main/scala/akka/stream/alpakka/s3/settings.scala

```diff
@@ -293,15 +409,17 @@ object S3Settings {
       s3RegionProvider: AwsRegionProvider,
       pathStyleAccess: Boolean,
       endpointUrl: Option[String],
-      listBucketApiVersion: ApiVersion
+      listBucketApiVersion: ApiVersion,
+      forwardProxy: Option[ForwardProxy]
```

Comment:
New fields shouldn't be added here.

Reply:
Ah, yes of course. Deprecated the old ones and added a new one without `proxy` and with `forwardProxy`.

Reply:
Don't offer a new one. The use `withForwardProxy` should be enough.

Reply:
The new one is still needed, as some attributes do not have default values. So I have changed the new one to only take those that do not have default values.